### PR TITLE
DDF-4173 Adding MutlLine/MultiPoly Rendering Support for OL

### PR DESCRIPTION
**Pull request by Schachte**
#### What does this PR do?
- Incorporates OL 2D rendering support for multi-geos. 
- Facilitates direct geo-removal in the openlayers map API 
- Modifies location old to support different multi-types
#### Who is reviewing it? 

@adimka 
@bdeining 
@djblue 
@vinamartin

#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Ask 2 committers to review/merge the PR and tag them here.
@bdeining 
@djblue 

#### What are the relevant tickets?
[DDF-4173](https://codice.atlassian.net/browse/DDF-4173)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js [hunk 1]
```diff
@@ -581,6 +581,68 @@ module.exports = function OpenlayersMap(
       })
       shapes.push(line)
     },
+    showMultiLineShape: function(locationModel) {
+      var lineObject = locationModel.get('multiline').map(line => {
+        return line.map(coords => {
```
Comment (adimka) [inline ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js]: ✏️ 1 line lambdas can exclude return and curly braces.

File: ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js [hunk 2]
```diff
@@ -581,6 +581,68 @@ module.exports = function OpenlayersMap(
...
+          return convertPointCoordinate(coords)
+        })
+      })
+
+      var feature = new Openlayers.Feature({
```
Comment (adimka) [inline ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js]: Here down is the same in both of these functions - we could break this out into a render vectorShape that takes the type as a param reduce code and prevent additional copypastes.

File: ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js [hunk 3]
```diff
@@ -581,6 +581,68 @@ module.exports = function OpenlayersMap(
...
+        source: vectorSource,
+      })
+
+      map.addLayer(vectorLayer)
+      overlays[locationModel.cid] = vectorLayer
+
+      return vectorLayer
+    },
+    destroyShape: function(cid) {
+      shapes.forEach(function(shape, i) {
```
Comment (adimka) [inline ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js]: ❓ @djblue, is there a more idomatic way to do this?

**Comment by bdeining** [comment]
build now

File: ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js [hunk 4]
```diff
@@ -581,6 +581,57 @@ module.exports = function OpenlayersMap(
...
+      let vectorLayer = new Openlayers.layer.Vector({
+        source: vectorSource,
+      })
+
+      map.addLayer(vectorLayer)
+      overlays[locationModel.cid] = vectorLayer
+
+      return vectorLayer
+    },
+    destroyShape: function(cid) {
```
Comment (gordocanchola) [inline ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js]: ❓ Is `destroyShape` used anywhere?

**Comment by Schachte** [comment]
build now

**Pull request merged**